crypto/low_power.h

```diff
@@ -0,0 +1,6 @@
+#ifndef XMR_STAK_CPU_LOW_POWER_H
```

Comment:
Please use `#pragma once` this is compatible to all compiler and much cleaner than guards per file via precompiler defines.